src/js/plugins/stellar/plugin.js

```diff
@@ -64,7 +64,7 @@ const transformAmount = amount => new BigNumber(amount).times(10000000).toString
 const transformMemo = memo => {
     switch (memo.type) {
         case StellarSdk.MemoText:
-            return { type: 1, text: memo.value };
+            return { type: 1, text: memo.value.toString('utf-8') };
```

Comment:
could you please past link to docs about memo type?

Reply:
https://github.com/stellar/js-stellar-base/blob/master/types/index.d.ts#L204

Reply:
and can't this happen to be `string` type? in that case toString method call would fail with 'utf-8' argument.

Reply:
I'm not sure what you are referring to, but hope this helps

https://github.com/trezor/trezor-firmware/pull/1745#discussion_r683263945

Reply:
```
const StellarSDK = require("stellar-sdk")
// text memo
// memo value: "hello"
let tx = StellarSDK.TransactionBuilder.fromXDR("AAAAAgAAAADQ7sFU88KfOExfripsgTcWhrq3exqUlYC1x5Qphkv+kwAAF3AAAACNbLKV6gAAAAEAAAAAAAAAAAAAAABjlFcZAAAAAQAAAAVoZWxsbwAAAAAAAAEAAAAAAAAABAAAAAAAAAABVVNEQwAAAAA7mRE4Dv6Yi6CokA6xz+RPNm99vpRr7QdyQPf2JN8VxQAAAAABMS0AAAAAAQAAAAEAAAAAAAAAAA==", StellarSDK.Networks.PUBLIC)
console.log(tx.memo.value instanceof Buffer) // true
console.log(tx.memo.value.toString("utf-8"))
```

Reply:
Yep all good. Sorry I got a bit confused. toString method works in the same way for both String and Buffer here.